Pod/Classes/OTSpanContext.h

```diff
@@ -0,0 +1,32 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * OTSpanContext represents Span state that must propagate to descendant Spans and across process boundaries.
+ *
+ * OTSpanContext is logically divided into two pieces: the user-level "Baggage" (see set_baggage_item and get_baggage_item) that propagates across Span boundaries and any Tracer-implementation-specific fields that are needed to identify or otherwise contextualize the associated Span instance (e.g., a <trace_id, span_id, sampled> tuple).
```

Comment:
nit: are really long lines like this an Objective-C convention? Glancing at the [NSObject source](http://opensource.apple.com//source/objc4/objc4-680/runtime/NSObject.mm), this seems not to be the case, but I certainly am not an expert on accepted Objective-C conventions.